edit-post/store/effects.js

```diff
@@ -27,7 +27,7 @@ const effects = {
 		}
 
 		// Allow toggling metaboxes panels
-		window.postboxes.add_postbox_toggles( 'post' );
+		window.postboxes.add_postbox_toggles( action.postType );
```

Comment:
Is `getCurrentPostType( store.getState() )` not an option?

Reply:
Unfortunately no, getCurrentPostType is a selector from the editor store, here we are in the edit-post store and there is no information in this store referring postType.

Reply:
I guess we can use `wp.data.select( 'core/editor' ).getCurrentPostType()` :)

Reply:
Yes, we can do that, the code was updated. I had some fears that the initialization of the postType core/editor was after edit/post but that was not the case.